app/invoice/page.tsx

```diff
@@ -0,0 +1,18 @@
+import FileUpload from '@/components/invoice-upload';
```

Comment:
component name and export name are not logically matching.

Reply:
done!